### ISSUE TITLE
[Aztec + GB] cover-image block with multiline text gets duplicated on each open

### ISSUE THREAD

**Issue by mzorz** [wordpress-mobile/AztecEditor-Android#683]
### Expected
Aztec should not modify content

### Observed
When a Post containing a Gutenberg `cover-image` block type is opened on Aztec, it duplicates the content under certain conditions (it has to have 2 lines of text at least)

### Reproduced
1. If you only enter one line of text, it works OK.
2. If you enter two lines of text in GB web, like this:
```
<!-- wp:cover-image {"url":"https://selfhostedmario.mystagingwebsite.com/wp-content/uploads/2018/05/0_ZLf40wNK_GA4015r-8.png","id":350} -->
<div class="wp-block-cover-image has-background-dim" style="background-image:url(https://selfhostedmario.mystagingwebsite.com/wp-content/uploads/2018/05/0_ZLf40wNK_GA4015r-8.png)">
	<p class="wp-block-cover-image-text">one libe<br/>ywo kine<br/><br/></p>
</div>
<!-- /wp:cover-image -->
```
Then open it on Aztec, and switch to HTML, this is what gets displayed:

```
<!-- wp:cover-image {"url":"https://selfhostedmario.mystagingwebsite.com/wp-content/uploads/2018/05/0_ZLf40wNK_GA4015r-8.png","id":350} -->
<div class="wp-block-cover-image has-background-dim" style="background-image: url(https: //selfhostedmario.mystagingwebsite.com/wp-content/uploads/2018/05/0_ZLf40wNK_GA4015r-8.png)">
	<p class="wp-block-cover-image-text">one libe<br>ywo kine</p>
</div><br>
<p class="wp-block-cover-image-text"></p>
<!-- /wp:cover-image -->
```

If you keep opening the Post and switching to HTML, then leaving it, each time it duplicates the content... this is the result after 3 times:

```
<p></p>

<!-- wp:cover-image {"url":"https://selfhostedmario.mystagingwebsite.com/wp-content/uploads/2018/05/0_ZLf40wNK_GA4015r-8.png","id":350} -->
one libe<br><br>ywo kine
<!-- /wp:cover-image -->

<p></p>

<!-- wp:cover-image {"url":"https://selfhostedmario.mystagingwebsite.com/wp-content/uploads/2018/05/0_ZLf40wNK_GA4015r-8.png","id":350} -->
one libe<br><br>ywo kine
<!-- /wp:cover-image -->

<p></p>

<!-- wp:cover-image {"url":"https://selfhostedmario.mystagingwebsite.com/wp-content/uploads/2018/05/0_ZLf40wNK_GA4015r-8.png","id":350} -->
<p class="wp-block-cover-image-text">one libe<br><br>ywo kine</p>
<!-- /wp:cover-image -->

<p></p>
```

I still have to go to the deep root of it, but this should suffice for reporting and opening an issue to have a starting point where to debug and find a fix for.

#### Tested
[Google Pixel] on [8.0] with [1.3.4]

**Comment by mzorz** [comment]
test

**Comment by mzorz** [comment]
test 2

**Comment by mzorz** [comment]
test3

**Comment by mzorz** [comment]
Probably related https://github.com/wordpress-mobile/AztecEditor-Android/issues/688

**Comment by daniloercoli** [comment]
Fixed in #697
